app/models/mixins/read_only_mixin.rb

```diff
@@ -10,7 +10,7 @@ module ReadOnlyMixin
   def reject_if_read_only
     if read_only? && !EvmDatabase.seeding?
       model_name = self.class.name.pluralize.underscore.tr('_', ' ').gsub('miq ', '')
-      errors.add(:base, _("Read only #{model_name} cannot be deleted."))
+      errors.add(:base, _("Read only %{model_name} cannot be deleted.") % model_name)
```

Comment:
Think this is wrong...needs to be:

```suggestion
      errors.add(:base, _("Read only %{model_name} cannot be deleted.") % {:model_name => model_name})
```